### DISCUSSION TITLE
Q: 126邮箱设置了SENDER_NAME但是发送人没有改变 || Q: 126 mailbox has set SENDER_NAME but the sender has not changed

### DISCUSSION THREAD

**Discussion by merept** [walinejs/waline#2077] [Q&A]
### Describe the bug | 问题描述

在vercel配置了`SENDER_NAME`环境变量，但是在接收到的邮件的发件人栏依旧是邮箱名

### Website URL | 问题网站

https://mere.pt/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

The `SENDER_NAME` environment variable is configured in vercel, but the sender column of the received email is still the mailbox name

### Website URL | Problem Website

https://mere.pt/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by kdyzm** [comment]
我也遇到了这个问题，不知道如何解决，我提了issue，但是作者似乎觉得这个问题是正常的

---
I also encountered this problem and didn’t know how to solve it. I raised an issue, but the author seemed to think this problem was normal

**Comment by MingTechpro** [comment]
> ### Describe the bug | 问题描述
> 在vercel配置了`SENDER_NAME`环境变量，但是在接收到的邮件的发件人栏依旧是邮箱名
> 
> ### Website URL | 问题网站
> https://mere.pt/
> 
> ### Where your waline deploy? | 服务部署在哪里？
> Vercel (Default)
> 
> ### Where your comment data store? | 数据存储在哪里？
> LeanCloud(https://leancloud.app)
> 
> ### Describe the bug | Problem description
> The `SENDER_NAME` environment variable is configured in vercel, but the sender column of the received email is still the mailbox name
> 
> ### Website URL | Problem Website
> https://mere.pt/
> 
> ### Where your waline deploy? | Where is the service deployed?
> Vercel (Default)
> 
> ### Where your comment data store? | Where is your comment data store?
> LeanCloud(https://leancloud.app)

查看是否添加这个环境变量 `SENDER_EMAIL` ：发邮件的邮箱号。例如：123456@qq.com
